cookbook/specs/fvm.json

```diff
@@ -5,21 +5,19 @@
         "specs": [
```

Comment:
missing APIs: AuthNew, AuthVerify, ChainCheckBlockstore, ChainDeleteObj, ChainExportRangeInternal, ChainHotGC, ChainPrune, ChainPutObj, ChainSetHead, CreateBackup, EthGetTransactionReceiptLimited, GasEstimateFeeCap, GasEstimateGasLimit, GasEstimateGasPremium, GasEstimateMessageGas, ID, LogAlerts, LogList, LogSetLevel

go to https://docs.filecoin.io/reference/json-rpc. The list above is until "Log". There are many missing APIs, add those